src/Microsoft.AspNetCore.Razor.Tasks/DotnetToolTask.cs

```diff
@@ -115,11 +112,11 @@ public override void Cancel()
             string commandLineCommands,
             out int result)
         {
-            CompilerServerLogger.Log("Server execution started.");
+            Log.LogMessage("Server execution started.");
             using (_razorServerCts = new CancellationTokenSource())
             {
-                CompilerServerLogger.Log($"CommandLine = '{commandLineCommands}'");
-                CompilerServerLogger.Log($"ServerResponseFile = '{responseFileCommands}'");
+                Log.LogMessage($"CommandLine = '{commandLineCommands}'");
+                Log.LogMessage($"ServerResponseFile = '{responseFileCommands}'");
```

Comment:
Does this show up in the msbuild log by default? we want this to only show up if you've asked for debug level logging

Reply:
It shouldn't. We set the importance conditionally here https://github.com/aspnet/Razor/blob/dev/src/Microsoft.AspNetCore.Razor.Tasks/DotnetToolTask.cs#L37